test/EndToEnd/ProjectBuildTests.cs

```diff
@@ -37,18 +37,10 @@ public void ItCanNewRestoreBuildRunCleanMSBuildProject()
             var runCommand = new RunCommand()
                 .WithWorkingDirectory(projectDirectory);
 
-            //  Set DOTNET_ROOT as workaround for https://github.com/dotnet/cli/issues/10196
```

Comment:
Did you consider just acquiring a 3.0 runtime to have these tests running? Or do you prefer this change instead?

Reply:
Considered it, but figured that was not representative of what this test is actually trying to do and potentially more to undo..